framework/components/ema-context-banners/ema-context-banners.twig

```diff
@@ -31,7 +31,7 @@
   <div class="ecl-container">
     <div class="ema-context-banner__wrapper">
       {% if _title is defined and _title is not empty %}
-      <h1 class="ecl-heading ecl-heading--ema-h1">{{ _title }}</h1>
```

Comment:
`.ecl-heading` had a `margin: 0 0 1.66rem` that is no longer present. The difference is small but present.